aviatrix/resource_aviatrix_spoke_transit_attachment.go

```diff
@@ -130,10 +190,88 @@ func resourceAviatrixSpokeTransitAttachmentRead(d *schema.ResourceData, meta int
 		}
 	}
 
+	if attachment.SpokeBgpEnabled {
+		transitGatewayPeering := &goaviatrix.TransitGatewayPeering{
+			TransitGatewayName1: spokeGwName,
+			TransitGatewayName2: transitGwName,
+		}
+
+		transitGatewayPeering, err = client.GetTransitGatewayPeeringDetails(transitGatewayPeering)
+		if err == goaviatrix.ErrNotFound {
+			d.SetId("")
```

Comment:
Should the resource be marked as gone here? If yes, can this information be returned in `client.GetSpokeTransitAttachment`?

Reply:
I think it should be marked as gone. bgpspoke-transit conn is just like transit peering, if it can't be found, it's gone.

bgpspoke-transit conn and spoke-transit conn are different, backend only returns info about bgpspoke-transit conn in API `get_inter_transit_gateway_peering_details`.